Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# AI building Old Gods Issue

## Developer changelog:
- The AI can no longer ignore `potential` block and build an Old God wonder upgrade without waking the Old God.
- Replaced `cthun_offmap_decisions_enabled_trigger` and `nzoth_offmap_decisions_enabled_trigger` by modifiers to prevent possible bugs.


# How to test:
Let the game run on observer mode for 30 years and check if the AI is building/had built old god wonder upgrade(you can make sure they did by checking their religion, country history and old god status).
